.editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# defaults
+[*]
+insert_final_newline = true     
+
+# 4 space indentation
+[*.{ini,py,py.tpl,rst}]
```

Comment:
We don't have any `ini` or `.tpl` or `.rst`  files currently in the project.

Reply:
Please cross check what all files do we have currently in the repo, and adapt accordingly